### PR TITLE
Fix go vet errors.

### PR DESCRIPTION
**Pull request by l2cup**
Fixes go vet errors so go vet passes.

### CODE REVIEW

File: ethereum/parity/types.go [hunk 1]
```diff
@@ -520,8 +520,8 @@ func Walk(vmt *VmTrace) []*VmState {
 type Action struct {
 	CallType        string          `json:"callType"`
 	Hash            *common.Hash    `json:"hash"`
-	ParentHash      *common.Hash    `json:"hash"`
-	TransactionHash *common.Hash    `json:"hash"`
+	ParentHash      *common.Hash    `json:"parent_hash"`
```
Comment (nebojsa94) [inline ethereum/parity/types.go:523]:
```suggestion
	ParentHash      *common.Hash    `json:"parentHash"`
```

File: ethereum/parity/types.go [hunk 2]
```diff
@@ -520,8 +520,8 @@ func Walk(vmt *VmTrace) []*VmState {
...
+	TransactionHash *common.Hash    `json:"transaction_hash"`
```
Comment (nebojsa94) [inline ethereum/parity/types.go:524]:
```suggestion
	TransactionHash *common.Hash    `json:"transactionHash"`
```

File: stacktrace/exception.go [hunk 3]
```diff
@@ -12,8 +12,8 @@ type StackFrame struct {
 	ContractName    string          `json:"name"`
 	Line            int             `json:"line"`
 
-	Code   string `json:"code",firestore:"-"`
-	Op     string `json:"op",firestore:"-"`
+	Code   string `json:"code"   firestore:"-"`
```
Comment (nebojsa94) [inline stacktrace/exception.go:15]: We can remove firestore tags.

**Review by nebojsa94 — CHANGES_REQUESTED** [review]

File: ethereum/parity/types.go (hunk 1 above)
Comment (l2cup) [reply ethereum/parity/types.go]: Fixed.

File: ethereum/parity/types.go (hunk 2 above)
Comment (l2cup) [reply ethereum/parity/types.go]: Fixed.

File: stacktrace/exception.go (hunk 3 above)
Comment (l2cup) [reply stacktrace/exception.go]: Removed.

**Pull request merged**